app/config.py

```diff
@@ -555,6 +561,8 @@ class Production(Config):
 
     CRONITOR_ENABLED = True
 
+    DVLA_API_BASE_URL = "https://driver-vehicle-licensing.api.gov.uk"
```

Comment:
These seem inconsistent with how we manage URLs generally. Should we just have `DVLA_API_BASE_URL` defined on the base config class with a default of the uat URL, and pass in an env var to each of deployed apps to set this correctly?

Reply:
hmmm yes, you're right, this is inconsistent. i thought it was how we did it elsewhere but checking now it's clearly not. 

With that said, I prefer having this in code - these aren't secret, it's overkill to hide them in a gpg encrypted pass store, and I think it's a nicer dev experience to be able to quickly check what URL our envs are using without having to access secrets. Thoughts?

Reply:
This is configuration, not code, so shouldn't require a deploy to change. I think, just because we happen to store all of our configuration gpg encrypted, it doesn't mean that non-secret config should be hard-coded instead.

I do agree it's nice to be able to see the URL easily, but I think that points to our gpg-encryption-for-all-config being the wrong thing rather than this being the right thing.

Reply:
Hmmmm

* In the current setup we have two options: define in cred store or define in code. We've gone with define in cred store for other URLs, but we do have other flags defined in config.py: feature flags, default "from number", bucket names, etc. In some other repos we have some URLs defined in config (admin's ASSSET_DOMAIN, template preview's LETTER_LOGO_URL), but we also have ones set in the environment. document-download-api also has its hostname defined in the manifest.yml.j2 and then passed through as an environment variable.
* In the current setup, I prefer having it in the repo than in gpg for the reasons I listed above, but i appreciate that's a personal view
* This story is not about changing the current setup. We might want to consider that in the future but there's probably less incentive to look before we move off of paas?

Ultimately the decision at hand is between two flawed practices that we currently have available. I think that across our whole codebase we're not very consistent anyway so that pushes me away from storing this in credentials, but i'm not against it if you think it's better

Reply:
Yep yep. As I say, just sharing my thoughts - not blocking you on this point.